code/modules/events/ghost_role/blob.dm

```diff
@@ -9,12 +9,6 @@
 	category = EVENT_CATEGORY_ENTITIES
 	description = "Spawns a new blob overmind."
 
-/datum/round_event_control/blob/can_spawn_event(players, allow_magic = FALSE, fake_check = FALSE) //MONKESTATION ADDITION: fake_check = FALSE
```

Comment:
`dont_spawn_near_roundend = TRUE` is just set in the event overrides file now for blob